### ISSUE TITLE
Filters "Reported sites" tweaks

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#349]
For the stats at https://www.blocked.org.uk/reported-sites :

(1) Add after "Reports Sent" "Total sites" "Sites unique to ISP" 

(2) Amend "Sites Unblocked" to "Sites verified as unblocked by this ISP"

(3) Add "All sites unblocked at this ISP"

(4) Add text field for the methodology so we can explain how and why we present these stats this way.

**Comment by JimKillock** [comment]
Duplicate of #279, moving commentary there
